src/main/java/com/bravo/user/service/UserService.java

```diff
@@ -48,6 +48,22 @@ public UserReadDto retrieve(final String id){
     return resourceMapper.convertUser(user);
   }
 
+  public List <UserReadDto> retrieveByName(
+      final String name,
+      final PageRequest pageRequest,
+      final HttpServletResponse httpResponse
+  ){
+    final UserSpecification specification = new UserSpecification(UserFilter.builder()
+        .name(name)
+        .build());
+    final Page<User> userPage = userRepository.findAll(specification, pageRequest);
+    final List<UserReadDto> users = resourceMapper.convertUsers(userPage.getContent());
+    LOGGER.info("found {} user(s)", users.size());
```

Comment:
Thank you for using {} notation :+1:  .   In my view, this is a debug and not an info though.

Reply:
This is an `info` in the existing methods.  Changing this to a debug should be done along with all of the others after a broader discussion on standards.